Package.swift

```diff
@@ -14,7 +14,7 @@ let package = Package(
         .library(name: "SmithyTestUtil", targets: ["SmithyTestUtil"])
     ],
     dependencies: [
-        .package(url: "https://github.com/awslabs/aws-crt-swift.git", from: "0.3.1"),
+        .package(url: "https://github.com/awslabs/aws-crt-swift.git", .exactItem("0.4.0")),
```

Comment:
why `exactItem` here? in aws-sdk-swift we use just `exact` https://github.com/awslabs/aws-sdk-swift/blob/0.5.0/Package.swift

Reply:
Changed to `.exact()`